src/lib/ParseApp.js

```diff
@@ -374,13 +374,17 @@ export default class ParseApp {
   }
 
   fetchPushSubscriberCount(audienceId, query) {
-    let path = '/apps/' + this.slug + '/dashboard_ajax/push_subscriber_count';
-    let urlsSeparator = '?';
-    if (query){
-      path += `?where=${encodeURI(JSON.stringify(query))}`;
-      urlsSeparator = '&';
+    let promise;
+    if (!query) {
+      promise = new Parse.Query('_Audience').get(audienceId, { useMasterKey: true }).then(function(audience) {
```

Comment:
we may wanna add caching for those counts as may be quite expensive on the DB.

Reply:
@flovilmart do we want to add in caching with this PR or do we want to handle this later on?

Reply:
Also if this runs okay on your end this would be great to get in. About time we should put up a dashboard release so we can get past #811 .

Reply:
Yep, caching would need more work, not required

Reply:
Ehhh, maybe in this case we should cache. Although we could take it as is it would be nice to not have to come back and do that later.